composer.json

```diff
@@ -31,7 +31,8 @@
         "phpunit/phpunit": "^8.0",
         "orchestra/testbench": "^3.7|^4.0|^5.0",
         "mheap/phpunit-github-actions-printer": "^1.2",
-        "friendsofphp/php-cs-fixer": "^2.16"
+        "friendsofphp/php-cs-fixer": "^2.16",
```

Comment:
Please move this dependency to the [php-cs-fixer-rules](https://github.com/lenderspender/php-cs-fixer-rules) package composer.json